test/live_server.rb

```diff
@@ -1,4 +1,5 @@
 require 'sinatra/base'
+load 'test/helper.rb'
```

Comment:
That might do it, but I still think you should bring in only what you need.
Maybe `big_string` could be moved into a module that can be used by both
test/helper and live_server?

On Wed, Aug 31, 2016 at 1:11 PM, Mattia <notifications@github.com> wrote:

> In test/live_server.rb
> <https://github.com/lostisland/faraday/pull/604#discussion_r77029751>:
>
> > @@ -1,4 +1,5 @@
> >  require 'sinatra/base'
> > +load 'test/helper.rb'
>
> Good point, if it is complaining is probably because that file gets loaded
> twice. Maybe changing load with require will fix the issue at requiring
> the same file twice only loads it once
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/lostisland/faraday/pull/604/files/f0ea17749ae5bb5777818efc0fa7afe711ecb7ae#r77029751>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAMIN115Z8_xL9DT5wfXQNOeZ6tJRTzgks5qlbW5gaJpZM4JxAhc>
> .
>